ra-optimisation/pom.xml

```diff
@@ -21,6 +21,7 @@
         <module>ra-optimisation-afs</module>
         <module>ra-optimisation-afs-local</module>
         <module>search-tree-rao</module>
+        <module>rao-api</module>
```

Comment:
I think modules names should be in alphabetical order

Reply:
done